docs/traffic-policy/concepts/actions.mdx

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 4
+title: Actions
+pagination_previous: traffic-policy/concepts/expressions
+---
+
+# Actions
+
+In Traffic Policy, actions define what happens when a [Rule](/traffic-policy/concepts/rules) on a [Phase](/traffic-policy/concepts/phases) conditions are met. Actions allow you to manipulate, route or manage traffic in various ways.
+
+Some actions can short-circuit a request and immediately return back a response, while others go to the next action in the chain.
```

Comment:
```suggestion
Most actions are executed sequentially as defined in the Traffic Policy. Some Actions can short-circuit a request and immediately return back a response, ignoring any subsequent Actions.
```

Reply:
should we link to those short circuiting actions?

Reply:
Later